critiquebrainz/frontend/views/moderators.py

```diff
@@ -0,0 +1,19 @@
+from flask import Blueprint, render_template
+from flask import current_app
+from critiquebrainz.db import users as db_users
+
+moderators_bp = Blueprint('moderators', __name__)
+
+
+@moderators_bp.route('/')
+def moderators():
+    admins = current_app.config['ADMINS']
+    db_admins = db_users.getusers_id(admins)
+    db_admins = [dict(_) for _ in db_admins]
```

Comment:
I would suggest to do this conversion in the database function and return dictionaries from there. See my other comment in that part.